src/oEmbed.php

```diff
@@ -9,7 +9,7 @@
 /**
  * Polyfill for older WP versions.
  */
-class oEmbed extends \WP_oEmbed {
+class OEmbed extends \WP_oEmbed {
```

Comment:
This just looks weird to me, and it actually breaks a "strict" rule of camelCasing. And we can't even be sure people don't already extend this in third-party commands.

I'd prefer whitelisting this one.
```suggestion
class oEmbed extends \WP_oEmbed { // phpcs:ignore PEAR.NamingConventions.ValidClassName.StartWithCapital
```

Reply:
Besides, you would have needed to adapt all the instantiations in the rest of the code if you go with the change.

Reply:
Class names are case-insensitive, so it doesn't matter if people extend this in third party commands, it will still work.

Would only be trouble-some if the class autoloader is case-sensitive (which it shouldn't be, but often is).

Reply:
Relying on case-insensitivity is usually a recipe for disaster when you go from development to production, as the optimized autloaders will fail where the development ones worked. Not a practice I want to rely on or encourage.

Reply:
Reverted the commit that renamed this class and whitelisted it in in the custom ruleset instead.